test/unit/update.js

```diff
@@ -65,14 +66,14 @@ exports['controller.update'] = {
         // `Version: ${build.version}\nPublished: ${published}\n${build.sha}\n`
         test.equal(this.basic.callCount, 2);
 
-        test.equal(
-          this.basic.getCall(0).args[0],
-          'Version: 0.0.1\nPublished: 8/11/2017, 11:01:57 PM\n9a85c84f5a03c715908921baaaa9e7397985bc7f\n'
-        );
-        test.equal(
-          this.basic.getCall(1).args[0],
-          'Version: 0.0.0\nPublished: 5/17/2017, 10:21:57 PM\nac4d8d8a5bfd671f7f174c2eaa258856bd82fe29\n'
-        );
+        const firstCall = this.basic.getCall(0).args[0];
+        test.equal(firstCall.startsWith('Version: 0.0.1'), true);
+        test.equal(firstCall.includes(builds[1].sha), true);
+
+        const secondCall = this.basic.getCall(1).args[0];
```

Comment:
Technically, this is not the "secondCall", it's the second call's args. I'm not going to block on that, so I'll leave it up to you if you want to change it.